src/main/java/com/github/onsdigital/babbage/content/client/ContentClient.java

```diff
@@ -384,8 +385,12 @@ private Map<String, String> getHeaders() {
         Map<String, String> cookies = (Map<String, String>) ThreadContext.getData("cookies");
         if (cookies != null) {
             HashMap<String, String> headers = new HashMap<String, String>();
-            headers.put(TOKEN_HEADER, cookies.get("access_token"));
-            return headers;
+            if (cookies.get("access_token") == null) {
+                throw new UnauthorizedException("User needs to login");
```

Comment:
what if user has an access_token but it is not authorised to see that content?
what if user has an access_token but it has expired?